EPlast/EPlast/gulpfile.js

```diff
@@ -28,6 +29,12 @@ gulp.task('less', function () {
         .pipe(gulp.dest(paths.webroot + 'css/lessCompiled'));
 });
 
+gulp.task("sass", function () {
+    return gulp.src('wwwroot/less/**/*.scss')
+        .pipe(sass())
+        .pipe(gulp.dest(paths.webroot + 'css/sassCompiled'));
+});
+
 gulp.task('build', gulp.parallel('scripts', 'less'));
```

Comment:
Please add sass task to the build otherwise it will not be executed when you build a solution